Assets/Scripts/Entities/Jackie.cs

```diff
@@ -72,19 +71,30 @@ override protected void DrawCard()
      */
     protected override void RenderHand()
     {
-        for (int i = 0; i < hand.Count; i++)
         {
-            hand[i].transform.SetParent(handContainer.transform, false);
-            hand[i].transform.position = Vector3.zero;
+            for (int i = 0; i < hand.Count; i++)
+            {
+                hand[i].transform.SetParent(handContainer.transform, false);
+                hand[i].transform.position = Vector3.zero;
 
-            float distanceToLeft = (float)(handContainer.rect.width / 2 - (i * cardWidth));
+                float distanceToLeft = (float)(handContainer.rect.width / 2 - (i * cardWidth));
 
-            float y = handContainer.transform.position.y;
-            Vector3 v = new Vector3(-distanceToLeft, y, -i);
-            hand[i].transform.position = v;
-            hand[i].transform.rotation = Quaternion.Euler(0, 0, -5);
+                float y = handContainer.transform.position.y;
+                Vector3 v = new Vector3(-distanceToLeft, y, -i);
+                hand[i].transform.position = v;
+                hand[i].transform.rotation = Quaternion.Euler(0, 0, -5);
+            }
+            RenderText();
+        }
+    }
+
+    // "unrenders" the hand
```

Comment:
This comment just restates the function name. Remove it or explain a bit more (if there's anything to say). See the comment on`RenderHand` for reference.

Reply:
I have had a bad history with function names not actually referring to the function's function. The emphasis exists to emphasise that unrendering ain't really unrendering.

Reply:
Okay. If you think there's something about this method that's worth documenting, you need to actually explain what it's doing. 

You've wrapped the word "unrender" in sarcastic quotes but you haven't told me what it actually does. The comment doesn't help me to understand your code.

```
/**
 * Moves the player's cards off-screen to hide them.
 * Note that this doesn't disable the card objects.
 */
```

Reply:
Now you see if this were complete this wouldn't be an issue. It's an INCOMPLETE PR. Ik it's bad practice. Clean-up occurs after completion (or during if you're so inclined).